android/src/app/organicmaps/widget/placepage/PlacePageLinksFragment.java

```diff
@@ -171,6 +171,13 @@ private String getLink(Metadata.MetadataType type)
     }
   }
 
+  private void openMail(Metadata.MetadataType type)
+  {
+    final String mail = getLink(type);
```

Comment:
Would it work without this helper method, if you patch getLink to return `mailto:<email>` and pass it into openUrl?

Reply:
We can't used openUrl to open mail,
openUrl used http scheme